src/views/Project/index.jsx

```diff
@@ -0,0 +1,247 @@
+import { Component } from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import ReactMarkdown from 'react-markdown';
+import { ApolloConsumer } from 'react-apollo';
+import _ from 'lodash';
+import gql from 'graphql-tag';
+import projects from '../../data/loader';
+import BugsTable from '../../components/BugsTable';
+import Issues from './issues.graphql';
+import Bugs from './bugs.graphql';
+
+class Project extends Component {
+  constructor(props) {
+    super(props);
+    const fileName = props.match.params.projectKey;
+    const projectInfo = projects[fileName];
+    // dictionary with repo as key and tag as value
+    const tmp = projectInfo.repositories
+      ? projectInfo.repositories.reduce(
+          (prev, cur) => ({ ...prev, ...cur }),
+          {}
+        )
+      : {};
+    // initial query search for all repo
+    const repoQuerySearch = ['state:open'];
+    // a dictionary of tag (string) and list of repository in that tag
+    const tagRepoDictionary = Object.keys(tmp).reduce((prev, key) => {
+      const curr = [
+        ...(prev[tmp[key]] || [...repoQuerySearch, `label:${tmp[key]}`]),
+        `repo:${key}`,
+      ];
+
+      return {
+        ...prev,
+        [tmp[key]]: curr,
+      };
+    }, {});
+    const tagRepoList = Object.entries(tagRepoDictionary).map(
+      ([key, value]) => ({
+        label: key,
+        query: value.join(' '),
+      })
+    );
+    // list of product with no component specified for Bugzilla Query
+    const productList = projectInfo.products.filter(
+      product => typeof product === 'string'
+    );
+    // list of product with its component specified for Bugzilla Query
+    const productTemp = projectInfo.products
+      .filter(product => typeof product !== 'string')
+      .reduce((prev, cur) => ({ ...prev, ...cur }), {});
+    const productComponentList = Object.entries(productTemp).map(
+      ([key, value]) => ({
+        products: [key],
+        components: value,
+      })
+    );
+
+    this.state = {
+      projectInfo,
+      tagRepoList,
+      error: false,
+      loading: true,
+      data: [],
+      productList,
+      productComponentList,
+    };
+  }
+  componentDidMount() {
+    const { tagRepoList } = this.state;
+
+    this.fetchGithubDataNext(tagRepoList);
+    this.fetchBugzillaDataNext();
+  }
+
+  fetchGithubDataNext(tagRepoList) {
```

Comment:
As discussed on IRC, I don't think we need to worry about pagination right now.

Reply:
Yeah its updated on the github PR